src/transformers/configuration_utils.py

```diff
@@ -536,7 +536,7 @@ def from_pretrained(
             force_download (`bool`, *optional*, defaults to `False`):
                 Whether or not to force to (re-)download the configuration files and override the cached versions if
                 they exist.
-            resume_download:
+            resume_download (defaults to `None`):
```

Comment:
It was decided in https://github.com/huggingface/transformers/pull/30620#discussion_r1588300600 that simply `resume_download:` is fine in this context given it's a deprecated and unused parameter. It defaults to `None` but more importantly, it is completely ignored by downstream library.